docs/builders.rst

```diff
@@ -163,3 +163,21 @@ or
 .. hint::
 
     As an alternative, you can set the config option :ref:`needs_build_needumls` to export the needumls files during each build.
+
+
+.. _needs_id_builder:
+
+needs_id
+--------
+.. versionadded:: 1.3.0
+
+The **needs_id** builder exports all found needs and selected filter results to a set json files of each need with the name is ``id`` of need.
+
+The build creates a folder called :ref:``needs_build_json_per_id_path`` and all file json of each need inside the given build-folder.
```

Comment:
Is there an example of a usecase which would require json per need?

Reply:
If you have >50.000 needs, the needs.json file can be quite fast > 50 MB.
Which is ok when built and used locally.

For instance JS-ocde (permalink or the viewer) would need to download the complete package, even if they only need the information for one specific need. And downloading and processing 50MB of code with each page-view may be quite fast annoying :)